src/main/Port.Adapter/UI/Maui/ViewModels/AddNeuronPermitViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using ei8.Avatar.Installer.Application.IdentityAccess;
+using ei8.Avatar.Installer.Common;
+using ei8.Avatar.Installer.Domain.Model.IdentityAccess;
+using neurUL.Common.Domain.Model;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ei8.Avatar.Installer.Port.Adapter.UI.Maui.ViewModels;
+
+public partial class AddNeuronPermitViewModel : EditAvatarViewModel
```

Comment:
Please try to unify AddNeuronPermitViewModel with NeuronPermitDetailsViewModel, and similar classes for RegionPermits and Users. Also please try to unify AddNeuronPermitPage with  NeuronPermitDetailsPage, and similar classes for RegionPermits and Users. A state enum variable can be used to differentiate when creating and editing records eg. Mode.AddNew & Mode.Edit etc. Unifying these components will reduce maintenance costs by minimizing (UI) redundancy.